package.json

```diff
@@ -42,15 +42,15 @@
   "devDependencies": {
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-json": "^4.0.2",
-    "@rollup/plugin-node-resolve": "^7.1.1",
+    "@rollup/plugin-node-resolve": "^8.0.0",
     "clap": "^2.0.1",
     "coveralls": "^3.0.9",
     "eslint": "^6.8.0",
     "json-to-ast": "^2.1.0",
     "mocha": "^5.2.0",
     "nyc": "^14.1.1",
     "rollup": "^1.32.1",
-    "rollup-plugin-terser": "^5.3.0"
+    "rollup-plugin-terser": "^6.0.1"
```

Comment:
6.0 doesn't support rollup before 2.0, and currently rollup below 2.0 is used.
Also 6.0 has dropped support for Node < 10, but css-tree supports Node 8.0+ at the moment.

Reply:
Ok, I see. So maybe this could be changed in the future.